### ISSUE TITLE
Missing some standard Windows equates

### ISSUE THREAD

**Issue by fenugrec** [NationalSecurityAgency/ghidra#1634]
In 9.1.2, pressing E (set Equate) on the constant 0x20019 only shows one value, where I would expect at least two others :

KEY_READ (0x20019)
KEY_EXECUTE (0x20019)
(see https://docs.microsoft.com/en-us/windows/win32/sysinfo/registry-key-security-and-access-rights )

But it does work for some other constants like HKEY_CURRENT_USER , so perhaps only a few headers are missing ?

**Comment by astrelsky** [comment]
Do the equate defines exist in the provided vs gdt? They should be named define_KEY_READ I think.

If they exist in the provided gdt then this is related to #1575

**Comment by fenugrec** [comment]
> Do the equate defines exist in the provided vs gdt?

No. I believe they're part of winreg.h , which is present in the Data Types list, and includes those HKEY_... defines, but none of the KEY_... defines.

Also tried _KEY_READ in the searchbox but as expected it didn't help.

**Comment by fenugrec** [comment]
My bad, it's actually defined in winnt.h (but still missing in gdt) :
https://github.com/wine-mirror/wine/blob/master/include/winnt.h#L5934

**Comment by astrelsky** [comment]
> > Do the equate defines exist in the provided vs gdt?
> 
> No. I believe they're part of winreg.h , which is present in the Data Types list, and includes those HKEY_... defines, but none of the KEY_... defines.
> 
> Also tried _KEY_READ in the searchbox but as expected it didn't help.

Ok thank you. I'm currently using my phone so I was unable to check.
